.codecov.yml

```diff
@@ -12,18 +12,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-coverage:
+codecov:
   notify:
-    after_n_builds: 30
+    wait_for_ci: yes
+
+coverage:
   status:
     patch: off
```

Comment:
Do it have to patch the project?


It didn't updated the GitHub check.

Reply:
Can you elaborate? I didn't get it.

Reply:
It seems that codecov updates the github check only once.

Reply:
I think that has nothing to do with the patch option... but anyway let me try turning it on and observe.

Reply:
Do you have any idea? Same behavior, but patch is green

Reply:
I searched around but didn't find anyone else reporting this problem. I am trying out a few settings at the moment. In the end, at least we can set the after_n_builds option back.